next.config.js

```diff
@@ -9,5 +9,13 @@ const withPWA = require('next-pwa')({
 
 module.exports = withPWA({
   reactStrictMode: true,
-  swcMinify: false
+  swcMinify: false,
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: 'agf-quartz.s3.amazonaws.com'
```

Comment:
Transformar em variabel ambiente
```suggestion
        hostname: process.env.IMAGES_HOSTNAME
```